wordpress-mobile/WordPress-iOS
# WordPressAuthenticator.framework: Mark I

### Details:
- Adds (an empty) WordPressAuthenticator subproject
- Adds a single Podfile dependency (CocoaLumberjack)
- Adds **CocoaLumberjack.swift** and **WPAuthenticatorLogging.h** (exact same code we've got for the WordPressShared.framework)
- Changes *zero* logic in WPiOS
- Hooks up WordPressAuthenticator.framework as a WPiOS dependency
- Hooks up WordPressUI + WordPressKit + WordPressShared as Authenticator dependencies

### To test:
1. Verify the main project builds correctly
2. Verify no unit test breaks

@diegoreymendez may i bug you with this one?.  Apologies about the PR size, most of it belongs to the new subproject's metadata.

**Thanks!!!**
